docker/Dockerfile

```
@@ -0,0 +1,36 @@
+# First stage: build the application
+FROM golang:1.13.0-alpine AS builder
+ARG GOARCH
+ARG GOARM
+
+WORKDIR /go/src/github.com/CESARBR/knot-babeltower
+COPY . .
+
+# Install build utilities and glide dependency manager
+RUN apk add --update --no-cache --virtual .build-deps \
+    make \
+    git \
+    curl
+RUN curl https://glide.sh/get | sh
+
+# Install project development tools and dependencies
+RUN go get github.com/ahmetb/govvv
+RUN make deps
+
+# Build the application
+RUN make bin
+
+# Remove build dependencies
+RUN apk del .build-deps
+
+# Second stage: create the entrypoint to the application binary generated in the previous stage
+FROM alpine:3.10
```

Is there a strong reason for us to use alpine here? If not, we could use `scratch` instead! The reason for this is that we will already have a static version of our application to run in it and it `scratch` in an empty image (meaning the final image size would be really small) :point_down: 

```suggestion
FROM scratch
```

If the tools present in alpine are useful for us, then we should keep alpine, but I suggest upgrading it :smile: 

```suggestion
FROM alpine:3.11
```

---

I think there isn't but I will check. Great suggestion! 😄

---

It works!